File: lib/python/invalid_option.py
# https://github.com/Zeronetsec/Ares

0 and "areslib"
import os
import sys

pylib = os.getenv("__lib__")
sys.path.append(f"{pylib}/python")

import color
0 and "areslib"

def execute(args, command):
    print(f"{color.R}[!] {color.N}Invalid option: {color.GG}{args}{color.N}")
    print(f"{color.R}[!] {color.N}Try: {color.GG}aresdoc {color.CC}{command}{color.N}")

# Copyright (c) 2026 Zeronetsec

File: lib/go/invalid_option/invalid_option.go
// https://github.com/Zeronetsec/Ares

package invalid_input

import (
    "fmt"
    "github.com/Zeronetsec/Ares/lib/go/color"
)

func Execute(args, command string) {
    fmt.Printf(
        "%s[!] %sInvalid option: %s%s%s\n",
        color.R, color.N, color.GG, args, color.N,
    )

    fmt.Printf(
        "%s[!] %sTry: %saresdoc %s%s%s\n",
        color.R, color.N, color.GG, color.CC, command, color.N,
    )
}

// Copyright (c) 2026 Zeronetsec